libcodechecker/analyze/statistics_collector.py

```diff
@@ -209,16 +205,20 @@ class ReturnValueCollector(object):
     # Checker name which runs the analysis.
     checker_analyze = 'statisticsbased.UncheckedReturnValue'
 
-    def __init__(self):
+    def __init__(self, stats_min_sample_count=10,
+                 stats_relevance_threshold=0.85):
```

Comment:
These default values are already provided at the argument parser at the outermost level. These are redundant.